Readd series notes (#1561)
jellyfin/jellyfin.org: docs/general/server/media/shows.md

Series Name (year) [external id]
```

> The series name should match the name of the series as listed by your metadata provider, however certain characters cannot be used as they are reserved by jellyfin such as dashes `-` and asterics `*`. For example with the series called `M*A*S*H`, use `MASH` or with `Stargate SG-1`, use `Stargate SG1` as the series name instead. This also applies to episode names as described below.

Reviewer: The don't use special characters is already mentioned right before the organization section, why again shortly after?
<img width="901" height="455" alt="截圖 2025-09-22 下午3 51 37" src="https://github.com/user-attachments/assets/3949a67b-b8d5-44d0-b632-ff9516f5f382" />
<img width="924" height="599" alt="截圖 2025-09-22 下午3 51 31" src="https://github.com/user-attachments/assets/e58fb9ca-87dd-4fda-b21e-b23694cff393" />

The `year` and `external id` fields are optional, but they will help identify media more reliably.

- Example with name only: `Jellyfin Documentary.mkv`
- Example with year: `Jellyfin Documentary (2030)`
- Example with external id: `Jellyfin Documentary [imdbid-tt00000000]`
- Example with both year and external id: `Jellyfin Documentary (2030) [imdbid-tt00000000]`

> The Season folders should be named `Season *`, with `*` being any number. Do not abbriviate the `Season` name to `S01` or `SE01` but name the folders `Season`. For the best results, please pad the season number with `0`s at the front to make sure each entry has the same number of digits. For example: `Season 5` -> `Season 05`. Also do not mix Season folders with episodes on the Series folder.

Reviewer 2: ```suggestion
The Season folders should be named `Season *`, with `*` being any number. Do not abbreviate the `Season` name to `S01` or `SE01` but name the folders `Season`. For the best results, please pad the season number with `0`s at the front to make sure each entry has the same number of digits. For example: `Season 5` -> `Season 05`. Also do not mix Season folders with episodes in the Shows folder.
```

I am not a fan of the wording in the first half: "Do not change the name from THIS  (e.g. to this or that) instead make sure to name it THIS" - I think this is very repetitive. Especially since the sentence right in front already tells you to name it "THIS". 

Additionally I do not like how the last sentence makes it seem like we allow Episodes in the `Shows` folder at all - as far as I can tell that is never mentioned anywhere above.
